Mathlib/RingTheory/Subring/Basic.lean

```diff
@@ -1234,6 +1234,10 @@ theorem range_top_of_surjective (f : R →+* S) (hf : Function.Surjective f) :
   range_top_iff_surjective.2 hf
 #align ring_hom.range_top_of_surjective RingHom.range_top_of_surjective
 
+section eqLocus
+
+variable {S : Type v} [Semiring S]
```

Comment:
Is this a change? Can you either revert it, mention it in the commit message, or move it to it's own PR?

Reply:
Yeah I'll add it to the commit message. It's a rather trivial change (Ring to Semiring). I simply saw an opportunity to generalize `eqLocusField` which requires generalizing this too.